File: ruby/bubble_sort.rb
def bubble_sort(arr)
    (0...arr.length - 1).each do |i|
        if arr[i] > arr[i + 1]
            arr[i], arr[i + 1] = arr[i + 1], arr[i]
        end
    end

    arr
end

arr = [2, 8, 5, 2, 6]
p bubble_sort(arr)

File: ruby/bubble_sort.js
function bubble_sort (arr) {
    for (let i = 0; i < arr.length; i++) {
        for (let j = 0; j < arr.length; j++) {
            if (arr[j] > arr[j + 1]) {
                [arr[j], arr[j + 1]] = [arr[j + 1], arr[j]];
            }
        }
    }
    console.log(arr);
}

let arr = [234, 53, 55, 3, 2, 1, 40];

bubble_sort(arr);